config/samples/tyk_v1alpha1_operatorcontext.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: OperatorContext
+metadata:
+  name: operatorcontext-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
Can we delete these files - and stop the framework scaffolding them?

Reply:
done b73d21b